data/test-by-folder.json

```
@@ -3284,6 +3284,17 @@
       "Mobile V2: Messaging - Save and Unsave Message"
     ]
   },
+  {
+    "folder": "mobile-v2/mobile-draft",
+    "tests": [
+      "Create Scheduled message in a Channel.",
+      "Deleting a scheduled messages in mobile.",
```

I think since it's in mobile-v2 folder, can we safely say they are all mobile?

```suggestion
      "Deleting a scheduled message.",
```

---

Yes Rahim